nxdrive/direct_edit.py

```diff
@@ -290,6 +291,10 @@ def _download(
 
             pair = engine.dao.get_valid_duplicate_file(blob.digest)
 
+        # Remove output file if already exist. See NXDRIVE-2112
```

Comment:
```suggestion
        # Remove the eventual temporary file. We do not want to be able to resume an
        # old download because of several issues and does not make sens for that feature.
        # See NXDRIVE-2112 and NXDRIVE-2116 for more context.
```